task/tekton-prune/0.1/tekton-prune.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: tekton-prune
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: tekton, misc
+    tekton.dev/displayName: "tekton objects cleanup"
+spec:
+  description: >-
+    This task help you cleanup old tekton objects.
+    .
+    This usually used to cleanup the Tekton *Runs objects but it can be used for
+    any other kubernetes objects.
+  params:
+    - name: object
+      type: string
+      description: The type of the tekton object to clean
+      default: ""
+    - name: max_days
+      type: string
+      description: Maximum number of days to keep.
+      default: ""
+    - name: max_number
+      type: string
+      description: Maximum number of object to keep.
+      default: ""
+    - name: labels
+      description: Labels to filter the tekton objects separated by a comma.
+      default: ""
+    - name: no_execute
+      type: string
+      description: Wether to execute the actual deletion, (yes or no)
+      default: "no"
+    - name: namespace
+      type: string
+      description: The target namespace
+      default: ""
+  steps:
+    - name: tekton-object-cleanups
+      image: alpine/k8s:1.18.2
```

Comment:
can we have the image name in the format `abc.io/foo/bar:1.0@digest` :sweat_smile: